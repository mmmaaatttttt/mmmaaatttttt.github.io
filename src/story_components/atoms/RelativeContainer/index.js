import styled, { css } from "styled-components";

const RelativeContainer = styled.div`
  position: relative;
`;

export default RelativeContainer;
