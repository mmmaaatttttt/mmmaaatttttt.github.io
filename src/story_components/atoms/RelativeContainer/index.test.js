import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import RelativeContainer from ".";

it("renders successfully", () => {
  shallow(<RelativeContainer />);
});

it("matches snapshot", () => {
  const wrapper = shallow(
    <RelativeContainer>
      <div>hello</div>
    </RelativeContainer>
  );
  expect(toJson(wrapper)).toMatchSnapshot();
});
