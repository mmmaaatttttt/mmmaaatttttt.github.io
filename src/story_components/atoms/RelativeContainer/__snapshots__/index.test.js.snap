// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  position: relative;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  <div>
    hello
  </div>
</StyledComponent>
`;
